model.js

```diff
@@ -1,17 +1,53 @@
+const fetchS3GeoJson = require('./lib/fetch-s3-geojson')
+const config = require('config')
+if (!config.koopProviderS3) throw new Error(`ERROR: "koopProviderS3" must be defined in your config.`)
+if (!config.koopProviderS3.stores) throw new Error(`ERROR: "koopProviderS3.stores" must be defined in your config.`)
+const stores = config.koopProviderS3.stores
+
 /**
  * Model constructor
  */
 function Model () {}
 
 /**
- * Fetch data from source.  Pass result or error to callback.
- * This is the only public function you need to implement on Model
- * @param {object} express request object
+ * Fetch data from S3.  Pass result or error to callback.
+ * @param {object} req express request object
  * @param {function} callback
  */
-Model.prototype.getData = function (req, callback) {
-  // stub
-  callback(null, {})
+Model.prototype.getData = async function (req, callback) {
```

Comment:
The `async` has no use in the function.

Reply:
I ended up reworking `getData` so that it uses `await`, so I have kept the `async`.